src/test/java/com/exasol/adapter/dialects/derby/DerbySqlDialect.java

```diff
@@ -19,9 +20,11 @@
 public class DerbySqlDialect extends AbstractSqlDialect {
     static final String NAME = "DERBY";
     private static final Capabilities CAPABILITIES = createCapabilityList();
-    private static final List<String> SUPPORTED_PROPERTIES = Arrays.asList(SQL_DIALECT_PROPERTY,
-            CONNECTION_NAME_PROPERTY, CATALOG_NAME_PROPERTY, SCHEMA_NAME_PROPERTY, TABLE_FILTER_PROPERTY,
-            EXCLUDED_CAPABILITIES_PROPERTY, DEBUG_ADDRESS_PROPERTY, LOG_LEVEL_PROPERTY);
+    private static final List<String> SUPPORTED_PROPERTIES = createSupportedPropertiesList();
```

Comment:
Make this an instance field and pull it up to the base class.
Remove side effect from `add...`.
Don't override `getCapabilities` anymore.

Reply:
fixed